File: Coroutines/app/src/main/java/com/kantek/coroutines/models/LoginResponseBody.java
package com.kantek.coroutines.models;

import org.simpleframework.xml.Element;
import org.simpleframework.xml.Root;

@Root(name = "Body", strict = false)
public class LoginResponseBody {
    @Element(name = "loginResponse", required = false)
    private LoginResponse response;

    public LoginResponse getResponse() {
        return response;
    }

    public void setResponse(LoginResponse response) {
        this.response = response;
    }
}

File: Coroutines/app/src/main/java/com/kantek/coroutines/models/LoginResponseBody.kt
package com.kantek.coroutines.models

import org.simpleframework.xml.Element
import org.simpleframework.xml.Root

@Root(name = "Body", strict = false)
class LoginResponseBody {
    @get:Element(name = "loginResponse", required = false)
    @set:Element(name = "loginResponse", required = false)
    var response: LoginResponse? = null
}
